### PR TITLE
MIC-316: IaR - Decision: Conclusion

### PR DESCRIPTION
**Pull request by JackReeveMoJ**

### CODE REVIEW

File: server/routes/journeys/record-decision/conclusion/controller.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      const nfa = items[nfaIndex]!
+      items.splice(nfaIndex, 1)
+      items.push(nfa)
+    }
+    const opeIndex = items.findIndex(o => o.value === 'OPE')
+    if (opeIndex > -1) {
+      items.splice(opeIndex, 1)
+    }
+    return items
+  }
```
Comment (bill-moj) [inline server/routes/journeys/record-decision/conclusion/controller.ts:33]: i think this is a repeat of what was in /screen ?
can this be an exported function to be shared by both controller?

Comment (JackReeveMoJ) [reply server/routes/journeys/record-decision/conclusion/controller.ts:33]: not entirely, we're also getting rid of one of the options here too

File: server/routes/journeys/record-decision/conclusion/tests.cy.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    cy.findByRole('heading', { name: 'Investigation decision' }).should('be.visible')
+
+    cy.findByText('Record a CSIP investigation decision').should('be.visible')
+
+    cy.findByRole('group', { name: 'What’s the conclusion of the CSIP investigation?' }).should('be.visible')
+
+    cy.findByText('Include the name and role of the staff member signing off on the decision.').should('be.visible')
+
+    cy.findByRole('textbox', { name: 'Describe the reasons for the decision' }).should('be.visible')
+  }
```
Comment (bill-moj) [inline server/routes/journeys/record-decision/conclusion/tests.cy.ts:57]: might add a test for the back url too

File: server/routes/journeys/record-decision/conclusion/tests.cy.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubGetPrisoner')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubComponents')
+    cy.task('stubCsipRecordGetSuccess')
+    cy.task('stubDecisionSignerRoles')
```
[the remark this answers is not in the document]
Comment (JackReeveMoJ) [reply server/routes/journeys/record-decision/conclusion/tests.cy.ts:23]: awesome!

**Review by bill-moj — APPROVED** [review]

**Pull request merged**